packages/database/src/runPostMigration.js

```diff
@@ -86,6 +86,12 @@ export const runPostMigration = async driver => {
     ),
   );
 
+  // Refresh analytics in case they've been impacted by migrations
+  const start = Date.now();
+  await driver.runSql(`SELECT mv$refreshMaterializedView('analytics', 'public', true);`);
```

Comment:
I wanted to use `AnalyticsRefresher`'s static invokation to do this, however it relies on a `TupaiaDatabase` object and the `driver` object we have here doesn't support the `executeSql` function. I didn't really wanna change things in the `AnalyticsRefresher` so we have a small amount of code duplication here instead.